maintenance/liftOver_assemblies_dm3TOdm6.py

```diff
@@ -0,0 +1,538 @@
+"""
+Code for liftOver between assemblies
+tailored for dm3 to dm6
+"""
+
+import sys, os, glob
+import argparse
+import multiprocessing as mp
+import subprocess as sp
+import shlex
+import numpy as np
+import pandas as pd
+import tarfile
+import urllib.request
+import shutil
+from datetime import date
+import matplotlib.pyplot as plt
+from matplotlib import gridspec
+
+
+def progress_bar(percent, barLen = 50):
```

Comment:
This looks like a near-identical copy of the liftover_assemblies.py file. Having multiple copies of the same code is almost never a good idea, and will be a significant maintenance burden over time (especially if we have lots of species - consider what  happens if we find a small bug in the liftover code, which has been copied n times).

How about we have a file ``liftover.py`` which provides the basic utilities that you're using here like ``hapmap_to_bed`` etc, and then the code specific to each liftover that we do is in its own python file that imports ``liftover``?

Reply:
That is, basically just the ``main`` functions which you currently have?